src/app.js

```diff
@@ -520,15 +520,7 @@ var run = function () {
     } else {
       url = 'https://ethereum.github.io/solc-bin/bin/' + version;
     }
-    var isFirefox = typeof InstallTrigger !== 'undefined';
```

Comment:
No, it isn't: `Worker error: undefined`.

But the other worker issue I cannot reproduce.